Functions/Administration/xASL_adm_SortDicomToFolders.m

```diff
@@ -0,0 +1,117 @@
+function xASL_adm_SortDicomToFolders(pathDICOM, nDirLayers, bUseDCMTK, bVerbose)
```

Comment:
What is the difference between this function and ConvertDicomFolderStructure?

And why is this one in Adm and the other in WIP?

Reply:
this is the same function :) I just wanted to get rid of "CarefulSlow" and wanted a better description - we are sorting files rather than converting them. And also, I wanted to clean this and move to the main functions instead of keeping it in WIP.

Feel free to suggest a better name. Though I don't thing "Convert" was a good one.